### ISSUE TITLE
Request to Jackett server failed / error

### ISSUE THREAD

**Issue by AlexMihe** [Jackett/Jackett#1685]
Hello,

Since I've updated to the pre-release, now on 0.8.67.0 I always get the two errors:
- "Request to Jackett server failed" when I try to add an Indexer
- "Error testing indexer, request to Jackett server error" when I want to test indexers

I've added a copy of the logs below.

I'm on QNAP with the app from "Guerithault Stephane (QoolBox)" version 0.7.1437

Thank you for your help !

**Edit:** I just installed it on a container from linuxserver.io and the issues remains

-------------------------------

When | Level | Message
-- | -- | --
Aug 18, 2017 11:31:48 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Logs()'
Aug 18, 2017 11:31:48 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Logs()'
Aug 18, 2017 11:31:48 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 18, 2017 11:31:48 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Logs()'
Aug 18, 2017 11:31:48 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 18, 2017 11:31:48 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 18, 2017 11:31:48 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 18, 2017 11:31:48 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 18, 2017 11:31:48 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Logs()'
Aug 18, 2017 11:31:48 AM | Info | ApiControllerActionSelector SelectAction
Aug 18, 2017 11:31:48 AM | Info | ServerConfigurationController ExecuteAsync
Aug 18, 2017 11:31:48 AM | Info | HttpControllerDescriptor CreateController Jackett.Controllers.V20.ServerConfigurationController
Aug 18, 2017 11:31:48 AM | Info | DefaultHttpControllerActivator Create Jackett.Controllers.V20.ServerConfigurationController
Aug 18, 2017 11:31:48 AM | Info | DefaultHttpControllerActivator Create
Aug 18, 2017 11:31:48 AM | Info | HttpControllerDescriptor CreateController
Aug 18, 2017 11:31:48 AM | Info | DefaultHttpControllerSelector SelectController ServerConfiguration
Aug 18, 2017 11:31:48 AM | Info | DefaultHttpControllerSelector SelectController Route='action:logs,controller:ServerConfiguration'
Aug 18, 2017 11:31:48 AM | Info | xxxxxxxxxxxxxxxxx:9117/api/v2.0/server/logs?_=1503048614364
Aug 18, 2017 11:31:19 AM | Info | IndexerApiController Dispose
Aug 18, 2017 11:31:19 AM | Info | IndexerApiController Dispose
Aug 18, 2017 11:31:19 AM | Info | Content-type='application/json; charset=utf-8', content-length=6436
Aug 18, 2017 11:31:19 AM | Info | IndexerApiController ExecuteAsync
Aug 18, 2017 11:31:19 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 18, 2017 11:31:19 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 18, 2017 11:31:19 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync
Aug 18, 2017 11:31:19 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 18, 2017 11:31:19 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync
Aug 18, 2017 11:31:19 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 18, 2017 11:31:19 AM | Error | ApiControllerActionInvoker InvokeActionAsync
Aug 18, 2017 11:31:19 AM | Error | ReflectedHttpActionDescriptor ExecuteAsync
Aug 18, 2017 11:31:19 AM | Debug | Validated Setting -- Browser Simulation enabled
Aug 18, 2017 11:31:19 AM | Debug | Validated Setting -- Max Pages => 4
Aug 18, 2017 11:31:19 AM | Debug | Validated Setting -- Password (auth) => xxxxxxx
Aug 18, 2017 11:31:19 AM | Debug | Validated Setting -- Username (auth) => xxxxxxxxx
Aug 18, 2017 11:31:19 AM | Debug | Validating Settings ...
Aug 18, 2017 11:31:19 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 18, 2017 11:31:19 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 18, 2017 11:31:19 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync
Aug 18, 2017 11:31:19 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 18, 2017 11:31:19 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 18, 2017 11:31:19 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 18, 2017 11:31:19 AM | Info | HttpActionBinding ExecuteBindingAsync Model state is valid. Values: config=Jackett.Models.DTO.ConfigItem[]
Aug 18, 2017 11:31:19 AM | Info | FormatterParameterBinding ExecuteBindingAsync Parameter 'config' bound to the value 'Jackett.Models.DTO.ConfigItem[]'
Aug 18, 2017 11:31:19 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Value read='Jackett.Models.DTO.ConfigItem[]'
Aug 18, 2017 11:31:19 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Type='ConfigItem[]', content-type='application/json'
Aug 18, 2017 11:31:19 AM | Info | FormatterParameterBinding ExecuteBindingAsync Binding parameter 'config'
Aug 18, 2017 11:31:19 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 18, 2017 11:31:19 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 18, 2017 11:31:19 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 18, 2017 11:31:19 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Config(ConfigItem[] config)'

**Comment by kaso17** [comment]
please make sure your browser cache is cleared (Ctrl+F5)

**Comment by AlexMihe** [comment]
It did work, for when I want to test indexers. Thanks !

But I still have the issue when I'm trying to add "Abnormal" as an indexer, I have the error: "Request to Jackett server failed"


Thanks !!!

**Comment by kaso17** [comment]
please provide an enhanced log

**Comment by AlexMihe** [comment]
**Please find it below:**


> When | Level | Message
> -- | -- | --
> Aug 18, 2017 09:49:18 PM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Logs()'
> Aug 18, 2017 09:49:18 PM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Logs()'
> Aug 18, 2017 09:49:18 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
> Aug 18, 2017 09:49:18 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Logs()'
> Aug 18, 2017 09:49:18 PM | Info | HttpActionBinding ExecuteBindingAsync
> Aug 18, 2017 09:49:18 PM | Info | HttpActionBinding ExecuteBindingAsync
> Aug 18, 2017 09:49:18 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
> Aug 18, 2017 09:49:18 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
> Aug 18, 2017 09:49:18 PM | Info | ApiControllerActionSelector SelectAction Selected action 'Logs()'
> Aug 18, 2017 09:49:18 PM | Info | ApiControllerActionSelector SelectAction
> Aug 18, 2017 09:49:18 PM | Info | ServerConfigurationController ExecuteAsync
> Aug 18, 2017 09:49:18 PM | Info | HttpControllerDescriptor CreateController Jackett.Controllers.V20.ServerConfigurationController
> Aug 18, 2017 09:49:18 PM | Info | DefaultHttpControllerActivator Create Jackett.Controllers.V20.ServerConfigurationController
> Aug 18, 2017 09:49:18 PM | Info | DefaultHttpControllerActivator Create
> Aug 18, 2017 09:49:18 PM | Info | HttpControllerDescriptor CreateController
> Aug 18, 2017 09:49:18 PM | Info | DefaultHttpControllerSelector SelectController ServerConfiguration
> Aug 18, 2017 09:49:18 PM | Info | DefaultHttpControllerSelector SelectController Route='action:logs,controller:ServerConfiguration'
> Aug 18, 2017 09:49:18 PM | Info | http://XXXXXXXXXXXXXXXXXX:9119/api/v2.0/server/logs?_=1503085700656
> Aug 18, 2017 09:49:06 PM | Info | IndexerApiController Dispose
> Aug 18, 2017 09:49:06 PM | Info | IndexerApiController Dispose
> Aug 18, 2017 09:49:06 PM | Info | Content-type='application/json; charset=utf-8', content-length=6436
> Aug 18, 2017 09:49:06 PM | Info | IndexerApiController ExecuteAsync
> Aug 18, 2017 09:49:06 PM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
> Aug 18, 2017 09:49:06 PM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
> Aug 18, 2017 09:49:06 PM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync
> Aug 18, 2017 09:49:06 PM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
> Aug 18, 2017 09:49:06 PM | Info | RequiresIndexerAttribute OnActionExecutedAsync
> Aug 18, 2017 09:49:06 PM | Info | RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
> Aug 18, 2017 09:49:06 PM | Error | ApiControllerActionInvoker InvokeActionAsync
> Aug 18, 2017 09:49:06 PM | Error | ReflectedHttpActionDescriptor ExecuteAsync
> Aug 18, 2017 09:49:06 PM | Debug | Validated Setting -- Browser Simulation enabled
> Aug 18, 2017 09:49:06 PM | Debug | Validated Setting -- Max Pages => 4
> Aug 18, 2017 09:49:06 PM | Debug | Validated Setting -- Password (auth) => XXXXXXXXXX
> Aug 18, 2017 09:49:06 PM | Debug | Validated Setting -- Username (auth) => XXXXXXX
> Aug 18, 2017 09:49:06 PM | Debug | Validating Settings ...
> Aug 18, 2017 09:49:06 PM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Config(config=Jackett.Models.DTO.ConfigItem[])'
> Aug 18, 2017 09:49:06 PM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Config(config=Jackett.Models.DTO.ConfigItem[])'
> Aug 18, 2017 09:49:06 PM | Info | RequiresIndexerAttribute OnActionExecutingAsync
> Aug 18, 2017 09:49:06 PM | Info | RequiresIndexerAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
> Aug 18, 2017 09:49:06 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
> Aug 18, 2017 09:49:06 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
> Aug 18, 2017 09:49:06 PM | Info | HttpActionBinding ExecuteBindingAsync Model state is valid. Values: config=Jackett.Models.DTO.ConfigItem[]
> Aug 18, 2017 09:49:06 PM | Info | FormatterParameterBinding ExecuteBindingAsync Parameter 'config' bound to the value 'Jackett.Models.DTO.ConfigItem[]'
> Aug 18, 2017 09:49:06 PM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Value read='Jackett.Models.DTO.ConfigItem[]'
> Aug 18, 2017 09:49:06 PM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Type='ConfigItem[]', content-type='application/json'
> Aug 18, 2017 09:49:06 PM | Info | FormatterParameterBinding ExecuteBindingAsync Binding parameter 'config'
> Aug 18, 2017 09:49:06 PM | Info | HttpActionBinding ExecuteBindingAsync
> Aug 18, 2017 09:49:06 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
> Aug 18, 2017 09:49:06 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
> Aug 18, 2017 09:49:06 PM | Info | ApiControllerActionSelector SelectAction Selected action 'Config(ConfigItem[] config)'

**Comment by Gabe323** [comment]
I've been getting this same error for the last month or so and stopped using Jackett. Updated and tried again but still there. I get the error with IPT and HDBits but Torrentleech added fine.

======IPT======

Aug 19, 2017 01:55:01 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Logs()'
-- | -- | --
Aug 19, 2017 01:55:01 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Logs()'
Aug 19, 2017 01:55:01 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 19, 2017 01:55:01 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Logs()'
Aug 19, 2017 01:55:01 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 01:55:01 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 01:55:01 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 01:55:01 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 01:55:01 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Logs()'
Aug 19, 2017 01:55:01 AM | Info | ApiControllerActionSelector SelectAction
Aug 19, 2017 01:55:01 AM | Info | ServerConfigurationController ExecuteAsync
Aug 19, 2017 01:55:01 AM | Info | HttpControllerDescriptor CreateController Jackett.Controllers.V20.ServerConfigurationController
Aug 19, 2017 01:55:01 AM | Info | DefaultHttpControllerActivator Create Jackett.Controllers.V20.ServerConfigurationController
Aug 19, 2017 01:55:01 AM | Info | DefaultHttpControllerActivator Create
Aug 19, 2017 01:55:01 AM | Info | HttpControllerDescriptor CreateController
Aug 19, 2017 01:55:01 AM | Info | DefaultHttpControllerSelector SelectController ServerConfiguration
Aug 19, 2017 01:55:01 AM | Info | DefaultHttpControllerSelector SelectController Route='action:logs,controller:ServerConfiguration'
Aug 19, 2017 01:55:01 AM | Info | http://xxx.xxx.xxx:35671/api/v2.0/server/logs?_=1503132505496
Aug 19, 2017 01:54:56 AM | Info | IndexerApiController Dispose
Aug 19, 2017 01:54:56 AM | Info | IndexerApiController Dispose
Aug 19, 2017 01:54:56 AM | Info | Content-type='application/json; charset=utf-8', content-length=927
Aug 19, 2017 01:54:56 AM | Info | IndexerApiController ExecuteAsync
Aug 19, 2017 01:54:56 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 19, 2017 01:54:56 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 19, 2017 01:54:56 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync
Aug 19, 2017 01:54:56 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 01:54:56 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync
Aug 19, 2017 01:54:56 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 01:54:56 AM | Error | ApiControllerActionInvoker InvokeActionAsync
Aug 19, 2017 01:54:56 AM | Error | ReflectedHttpActionDescriptor ExecuteAsync
Aug 19, 2017 01:54:56 AM | Debug | IWebClient: Returning OK => Captcha verification failed.<br>please try again!
Aug 19, 2017 01:54:55 AM | Debug | UnixLibCurlWebClient: Posting username=xxx&password=xxx-recaptcha-response=
Aug 19, 2017 01:54:55 AM | Debug | IWebClient.GetString(Url:https://iptorrents.com/take_login.php)
Aug 19, 2017 01:54:55 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 19, 2017 01:54:55 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 19, 2017 01:54:55 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync
Aug 19, 2017 01:54:55 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 01:54:55 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 19, 2017 01:54:55 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 01:54:55 AM | Info | HttpActionBinding ExecuteBindingAsync Model state is valid. Values: config=Jackett.Models.DTO.ConfigItem[]
Aug 19, 2017 01:54:55 AM | Info | FormatterParameterBinding ExecuteBindingAsync Parameter 'config' bound to the value 'Jackett.Models.DTO.ConfigItem[]'
Aug 19, 2017 01:54:55 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Value read='Jackett.Models.DTO.ConfigItem[]'
Aug 19, 2017 01:54:55 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Type='ConfigItem[]', content-type='application/json'
Aug 19, 2017 01:54:55 AM | Info | FormatterParameterBinding ExecuteBindingAsync Binding parameter 'config'
Aug 19, 2017 01:54:55 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 01:54:55 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 01:54:55 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 01:54:55 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Config(ConfigItem[] config)'
Aug 19, 2017 01:54:55 AM | Info | ApiControllerActionSelector SelectAction
Aug 19, 2017 01:54:55 AM | Info | IndexerApiController ExecuteAsync

=======HDBITS========


Aug 19, 2017 02:05:13 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Logs()'
-- | -- | --
Aug 19, 2017 02:05:13 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Logs()'
Aug 19, 2017 02:05:13 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 19, 2017 02:05:13 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Logs()'
Aug 19, 2017 02:05:13 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 02:05:13 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 02:05:13 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 02:05:13 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 02:05:13 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Logs()'
Aug 19, 2017 02:05:13 AM | Info | ApiControllerActionSelector SelectAction
Aug 19, 2017 02:05:13 AM | Info | ServerConfigurationController ExecuteAsync
Aug 19, 2017 02:05:13 AM | Info | HttpControllerDescriptor CreateController Jackett.Controllers.V20.ServerConfigurationController
Aug 19, 2017 02:05:13 AM | Info | DefaultHttpControllerActivator Create Jackett.Controllers.V20.ServerConfigurationController
Aug 19, 2017 02:05:13 AM | Info | DefaultHttpControllerActivator Create
Aug 19, 2017 02:05:13 AM | Info | HttpControllerDescriptor CreateController
Aug 19, 2017 02:05:13 AM | Info | DefaultHttpControllerSelector SelectController ServerConfiguration
Aug 19, 2017 02:05:13 AM | Info | DefaultHttpControllerSelector SelectController Route='action:logs,controller:ServerConfiguration'
Aug 19, 2017 02:05:13 AM | Info | http://xxx.xxx.xxx:35671/api/v2.0/server/logs?_=1503132505503
Aug 19, 2017 02:05:07 AM | Info | IndexerApiController Dispose
Aug 19, 2017 02:05:07 AM | Info | IndexerApiController Dispose
Aug 19, 2017 02:05:07 AM | Info | Content-type='application/json; charset=utf-8', content-length=1080
Aug 19, 2017 02:05:07 AM | Info | IndexerApiController ExecuteAsync
Aug 19, 2017 02:05:07 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 19, 2017 02:05:07 AM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Aug 19, 2017 02:05:07 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync
Aug 19, 2017 02:05:07 AM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 02:05:07 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync
Aug 19, 2017 02:05:07 AM | Info | RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 02:05:07 AM | Error | ApiControllerActionInvoker InvokeActionAsync
Aug 19, 2017 02:05:07 AM | Error | ReflectedHttpActionDescriptor ExecuteAsync
Aug 19, 2017 02:05:07 AM | Debug | IWebClient: Returning OK => 2885 bytes
Aug 19, 2017 02:05:07 AM | Debug | IWebClient.GetBytes(Url:https://hdbits.org/login?error=7&returnto=%2Fmy.php)
Aug 19, 2017 02:05:07 AM | Debug | IWebClient: Returning Redirect => https://hdbits.org/login?error=7&returnto=%2Fmy.php
Aug 19, 2017 02:05:06 AM | Debug | UnixLibCurlWebClient: Posting csrf=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1MDMxMzQ0NjcsInVhIjoiTW96aWxsYVwvNS4wIChYMTE7IExpbnV4IHg4Nl82NCkgQXBwbGVXZWJLaXRcLzUzNy4zNiAoS0hUTUwsIGxpa2UgR2Vja28pIFVidW50dSBDaHJvbWVcLzQ3LjAuMjUyNi43MyBTYWZhcmlcLzUzNy4zNiIsImlwIjoiMTkyLjEzMS40NC45MyJ9.xiucmvKQjIqsxAy0iYSBy7AnP7z9uMjnXwaVmB5Y478&uname=xxx&password=xxx&twostep_code=
Aug 19, 2017 02:05:06 AM | Debug | IWebClient.GetString(Url:https://hdbits.org/login/doLogin)
Aug 19, 2017 02:05:06 AM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 19, 2017 02:05:06 AM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Config(config=Jackett.Models.DTO.ConfigItem[])'
Aug 19, 2017 02:05:06 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync
Aug 19, 2017 02:05:06 AM | Info | RequiresIndexerAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 02:05:06 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Aug 19, 2017 02:05:06 AM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Aug 19, 2017 02:05:06 AM | Info | HttpActionBinding ExecuteBindingAsync Model state is valid. Values: config=Jackett.Models.DTO.ConfigItem[]
Aug 19, 2017 02:05:06 AM | Info | FormatterParameterBinding ExecuteBindingAsync Parameter 'config' bound to the value 'Jackett.Models.DTO.ConfigItem[]'
Aug 19, 2017 02:05:06 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Value read='Jackett.Models.DTO.ConfigItem[]'
Aug 19, 2017 02:05:06 AM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Type='ConfigItem[]', content-type='application/json'
Aug 19, 2017 02:05:06 AM | Info | FormatterParameterBinding ExecuteBindingAsync Binding parameter 'config'
Aug 19, 2017 02:05:06 AM | Info | HttpActionBinding ExecuteBindingAsync
Aug 19, 2017 02:05:06 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 02:05:06 AM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Aug 19, 2017 02:05:06 AM | Info | ApiControllerActionSelector SelectAction Selected action 'Config(ConfigItem[] config)'

**Comment by thebluepotato** [comment]
I'm having the same issue (currently on 0.8.55.0). My guess is that the error string isn't passed correctly to the notification. For example when entering the wrong login, I get the red notification "Request to Jackett server failed" and the enhanced log shows:

When | Level | Message
-- | -- | --
Aug 19, 2017 05:14:03 PM | Error | ApiControllerActionInvoker InvokeActionAsync
Aug 19, 2017 05:14:03 PM | Error | ReflectedHttpActionDescriptor ExecuteAsync

Since it's a controlled login fail, in my case it should say something along the lines of "Login failed, selector did not match" (or "Login failed, was redirected" on other trackers).

When a tracker test fails, I get "Error testing indexer, request to Jackett server error", with the exact two same errors in the enhanced logs.

In short: the bug here isn't the failing of the logins or the tests (i.e. they're a separate issue), but the actual error reason isn't correctly sent to the logs/notifications.

**Comment by AlexMihe** [comment]
I have the error: "Request to Jackett server failed" only with Xthor and HD-Forever

Would anyone have an idea ?

Thanks !

**Comment by JigSawFr** [comment]
@thebluepotato correct, I've just seen since the new API of Jackett, when an error occurred, a generic message is displayed, I'm investigating this error.

**Comment by JigSawFr** [comment]
Fixed in https://github.com/Jackett/Jackett/commit/26dd740a53997ee34a94a539a5fdf766a04c9f87
